src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -89,10 +89,11 @@ const useResolveStep = () => {
   const mergeCycleCountItems = (items) => {
     const duplicatedItems = _.groupBy(items,
       (item) => `${item.binLocation?.id}-${item?.inventoryItem?.lotNumber}`);
-    return Object.values(duplicatedItems).map((itemsToMerge) => {
+
+    return Object.values(duplicatedItems).flatMap((itemsToMerge) => {
```

Comment:
@SebastianLib can you explain the changes in this PR? What was the issue and how you are fixing that.